app/routes/_layout+/assets._index.tsx

```diff
@@ -343,11 +347,38 @@ const ListAssetContent = ({
               <span className="word-break mb-1 block font-medium">
                 {item.title}
               </span>
-              <div>
+              <div className="flex items-center gap-x-1">
                 <AssetStatusBadge
                   status={item.status}
                   availableToBook={item.availableToBook}
                 />
+                {item.custody?.template?.signatureRequired &&
+                  !item.custody.templateSigned && (
+                    <CustomTooltip
+                      content={
+                        <div className="flex flex-col gap-y-2 p-3">
+                          <span className="text-sm text-gray-700">
```

Comment:
I adjusted how the content is structured inside assetId, Can you also please adjust it here @rajdip-b

Reply:
Can you elaborate on this one? not sure what i need to change :/

Reply:
Just making sure the custom tooltip content is the same. It was quite some time ago so I dont remember exactly what changed. Just make sure the tooltip content is consistent with assetId route